tests/test_schema.py

```diff
@@ -1304,13 +1304,14 @@ class B(Document):
             'slm': [{'doc_type': 'A', 's': six.text_type(a2.s)},
                     {'doc_type': 'A', 's': six.text_type(a1.s)}]
         })
-        b.slm.sort(cmp=lambda x, y: cmp(x['s'].lower(), y['s'].lower()))
-        self.assertEqual([b.slm[0].s, b.slm[1].s], [a1.s, a2.s])
-        self.assertEqual(b._doc, {
-            'doc_type': 'B',
-            'slm': [{'doc_type': 'A', 's': six.text_type(a1.s)},
-                    {'doc_type': 'A', 's': six.text_type(a2.s)}]
-        })
+        if six.PY2:
```

Comment:
Why is this py2 only? Maybe add a comment here to explain that?

Reply:
https://github.com/dimagi/couchdbkit/pull/72